apps/docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,323 @@
+ * tocVideo: optional - a YouTube slug, displays the video above the table of contents
+ */
+export const meta = {
+  id: 'nextjs-server-components',
+  title: 'Supabase Auth with Next.js Server Components',
+  description:
+    'Authentication helpers for creating an authenticated Supabase client in Next.js 13 app directory Server Components.',
+  sidebar_label: 'Next.js Server Components',
+}
+
+// content
+You can use **Markdown syntax**.
+
+// required export
+export const Page = ({ children }) => <Layout meta={meta} children={children} />
+
+export default Page
+```
+
+The navigation is defined in [`NavigationMenu.constants.ts`](https://github.com/supabase/supabase/blob/master/apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts).
+
+Add an entry with the `name`, `url`, and (optional) `icon` for your page.
+
+## Content reuse
+
+If you copy the same content multiple times across different files, create a **partial** for content reuse instead. Partials are MDX files contained in [`apps/docs/components/MDX`](https://github.com/supabase/supabase/tree/master/apps/docs/components/MDX). They contain reusable snippets that can be inserted in multiple pages. For example, you can create a partial to define a common setup step for a group of tutorials.
+
+To use a partial, import it into your MDX file. You can also set up a partial to automatically import by including it in the `components` within [`apps/docs/components/index.tsx`](https://github.com/supabase/supabase/blob/master/apps/docs/components/index.tsx).
+
+## Components and elements
+
+Docs include normal Markdown elements such as lists, and custom components such as admonitions (callouts).
+
+Here are some guidelines for using elements:
+
+### Admonitions
+
+Admonitions (or callouts) draw reader attention to an important point or an aside. They highlight important information, but get less effective if they're overused.
+
+Use admonitions sparingly. Don't stack them on top of each other.
+
+Choose the appropriate `type` for your admonition:
+
+- **Danger** to warn the user about any missteps that could cause data loss or data leaks
+- **Deprecation** to notify the user about features that are (or will soon be) deprecated
+- **Caution** to warn about anything that could cause a bug or serious user inconvenience
+- **Tip** to point out helpful but optional actions
+- **Note** for anything else
+
+```
+<Admonition type="note" label="Optional label displays as title">
+
+Your content here
+
+</Admonition>
+```
+
+### Blockquotes
+
+Don't use blockquotes.
+
+### Code blocks
+
+Keep code lines short to avoid scrolling. For example, you can split long shell commands with `\`.
+
+- **JavaScript/TypeScript**
+
+  The `supabase` repo uses Prettier, which also formats JS/TS in code blocks. Your PR is blocked from merging if the Prettier check fails. Ensure that your code blocks are formatted by running `npm run format`, or by setting up auto-formatting in your IDE.
```

Comment:
```suggestion
The `supabase` repo uses Prettier, which also formats JS/TS in code blocks. Your PR is blocked from merging if the Prettier check fails. Ensure that your code blocks are formatted by running `npm run format`, or by setting up auto-formatting in your IDE.
```

Reply:
intentional to indent as part of list item